Yurii_Khomych/l_4_iterators_generators/HW_4_yuliia/Generators/generators.py

```diff
@@ -0,0 +1,27 @@
+# generator example 1
```

Comment:
але це не генератор)

Reply:
Corrected here https://github.com/YuriiKhomych/ITEA-advanced/pull/11 because of incorrect parent branch 
I'll remove this PR and branch as soon as https://github.com/YuriiKhomych/ITEA-advanced/pull/11 will be checked